sdl2-sys/build.rs

```diff
@@ -112,6 +114,111 @@ fn download_sdl2() -> PathBuf {
     sdl2_build_path
 }
 
+// apply patches to sdl2 source
+#[cfg(feature = "bundled")]
+fn patch_sdl2(sdl2_source_path: &Path) {
+    let patches: Vec<(&str, &'static str)> = vec![
+        ("SDL2-2.0.8-4234-mac-os-dylib-fix.patch",
```

Comment:
Add as a comment why this patch is necessary, please (we probably won't remember in 6months, and we won't even be able to know when to remove it if/when this is fixed)